src/SFA.DAS.Provider.PR.Web/Controllers/AddEmployer/SearchByPayeController.cs

```diff
@@ -98,6 +98,15 @@ public async Task<IActionResult> Index([FromRoute] int ukprn, SearchByPayeSubmit
             return RedirectToRoute(RouteNames.AddEmployerMultipleAccounts, new { ukprn });
         }
 
+        var isInEAS = relationshipsRequest?.Account != null;
```

Comment:
rename the var to read its purpose `hasExistingAccount`

Reply:
ok